server_admin/topics/login-settings/update-email-workflow.adoc

```diff
@@ -0,0 +1,25 @@
+[[_update-email-workflow]]
+
+=== Update Email Workflow (UpdateEmail)
+
+With this workflow, users will have to use an UPDATE_EMAIL action to change their own email address.
+
+The action is associated with a single email input form. If the realm has email verification disabled, this action will allow to update the email without verification. If the realm has email verification enabled, the action will send an email update action token to the new email address without changing the account email. Only the action token triggering will complete the email update.
+
+Applications are able to send their users to the email update form by leveraging UPDATE_EMAIL as an AIA (Application Initiated Action).
+
+ifeval::[{project_product}==true]
+:tech_feature_name: UpdateEmail
+:tech_feature_setting: -Dkeycloak.profile.feature.update_email=enabled
+:tech_feature_id: update-email
+include::../templates/techpreview.adoc[]
+endif::[]
+
+ifeval::[{project_community}==true]
+
+[IMPORTANT]
+====
+Please note that Update Email Workflow support is in development. Use this feature experimentally.
+====
+
+endif::[]
```

Comment:
Small note: Is it possible to add one small note about migration? Something like:

NOTE: If you enable this feature and you are migrating from previous version, you will need to register `Update Email` required action in your realms. Otherwise your users won't be able to update their email addresses.